libc/CMakeLists.txt

```diff
@@ -30,6 +37,11 @@ endif()
 # Default to C++17
 set(CMAKE_CXX_STANDARD 17)
 
+if(LIBC_BUILT_STANDALONE)
+  find_package(Python3 ${LLVM_MINIMUM_PYTHON_VERSION} REQUIRED
+    COMPONENTS Interpreter)
+endif()
```

Comment:
we also need `pyyaml` for newhdrgen. Not sure if that needs to be specified in the cmake, but adding a comment at least would be useful.

Reply:
I'll add a comment about it at least, I'm not sure how we could add a check inside CMake to ensure the module exists.